theano/tensor/nnet/abstract_conv.py

```diff
@@ -181,6 +183,8 @@ def get_conv_gradweights_shape(image_shape, top_shape,
 
     if filter_dilation is None:
         filter_dilation = np.ones(len(subsample), dtype='int')
+    if num_groups > 1 and len(subsample) == 2:
```

Comment:
Here, I would do something like:
if num_groups > 1:
    assert len(subsample) == 2
    nchan = nchan // num_groups

to get an error in the not supported case.